config/admins.txt

```diff
@@ -131,4 +131,5 @@ JStheguy = Game Master
 excessiveuseofcobby = Game Master
 Plizzard = Game Master
 octareenroon91 = Game Master
-Serpentarium = Game Master
+Serpentarium = Game Master
+Averagejoe82 = Game Master

```

Comment:
reeeeeeee

Reply:
Is this a bad thing? I thought the admin.txt on the GitHub didn’t matter

Reply:
Ye its fine

Reply:
yeah it doesn't matter

Reply:
Danke